src/m365/context/commands/context-option-add.spec.ts

```diff
@@ -0,0 +1,114 @@
+import * as assert from 'assert';
+import * as fs from 'fs';
+import * as sinon from 'sinon';
+import { Cli } from '../../../cli/Cli';
+import { Logger } from '../../../cli/Logger';
+import Command, { CommandError } from '../../../Command';
+import { telemetry } from '../../../telemetry';
+import { sinonUtil } from '../../../utils/sinonUtil';
+import commands from '../commands';
+const command: Command = require('./context-option-add');
+
+describe(commands.OPTION_ADD, () => {
+  let log: any[];
+  let logger: Logger;
+
+  before(() => {
+    sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
+  });
+
+  beforeEach(() => {
+    log = [];
+    logger = {
+      log: (msg: string) => {
+        log.push(msg);
+      },
+      logRaw: (msg: string) => {
+        log.push(msg);
+      },
+      logToStderr: (msg: string) => {
+        log.push(msg);
+      }
+    };
+  });
+
+  afterEach(() => {
+    sinonUtil.restore([
+      fs.existsSync,
+      fs.readFileSync,
+      fs.writeFileSync,
+      Cli.prompt
```

Comment:
We are not using `prompt` in the code. Hence it should not be available in the test case